common/src/main/java/earth/terrarium/hermes/api/defaults/HeadingTagElement.java

```diff
@@ -0,0 +1,51 @@
+package earth.terrarium.hermes.api.defaults;
+
+import com.teamresourceful.resourcefullib.client.CloseablePoseStack;
+import earth.terrarium.hermes.api.themes.Theme;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.network.chat.Component;
+import net.minecraft.util.FormattedCharSequence;
+
+import java.util.Map;
+
+public abstract class HeadingTagElement extends TextTagElement {
+
+    public final int SCALE;
```

Comment:
this should be lowercase as its not a constant

Reply:
Thank you for catching that. Fixed in 0d00f6a.